src/components/Home.js

```diff
@@ -0,0 +1,167 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faSearch, faVialVirus, faArrowRight, faAngleLeft, faEarthAfrica,
+} from '@fortawesome/free-solid-svg-icons';
+import { v4 as uuidv4 } from 'uuid';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData, setDetails } from '../redux/home/HomeSlice';
+import FullPageLoader from './Loader';
+import Details from './Alldetails';
+import './styles/Home.css';
+
+const Info = ({ confirmed, deaths }) => (
+  <div className="Info">
+    <h2 className="global">
+      {' '}
+      <i>
+        <FontAwesomeIcon icon={faEarthAfrica} />
+      </i>
+      Global Corona-virus  Census Cases
+    </h2>
+    <div className="Info-Wrapper">
+      <span className="span">
+        <p className="confirmed-text-home">CONFIRMED</p>
+        <p className="confirmed">{confirmed}</p>
+      </span>
+      <span>
+        <p className="deaths-text">DEATHS</p>
+        <p className="deaths">{deaths}</p>
+      </span>
+    </div>
+  </div>
+);
+
+const Section = ({ country, cases }) => {
+  const dispatch = useDispatch();
+  function handleClick() {
+    dispatch(setDetails(country));
+  }
+  return (
+    <div className="section">
+      <span className="country-info">
+        <p className="country-section-text">{country}</p>
+        <p className="cases">{cases}</p>
+      </span>
+      <button type="button" className="section-btn" onClick={() => handleClick()}>
+        <i>
+          <FontAwesomeIcon icon={faArrowRight} />
+        </i>
+      </button>
+
+    </div>
+  );
+};
+
+const Home = () => {
```

Comment:
-  Hi @hydaline-code, I think you're missing a very pertinent requirement of this project. It says "The Home page has a list of categories that could be filtered using given parameters, e.g. category name."  You were supposed to add a search input field on the home page to filter out a particular country(filter by country name). This is a very long list of Global Corona-virus Census Cases and a user may find it difficult to scroll and locate a country. It is recommended you kindly address this issue before merging your PR since it was not sported at the beginning.